index.html

```diff
@@ -248,9 +248,9 @@ <h1><span>La <strong>grève</strong>,</span> c’est <strong class="cool">cool</
 
         <section>
             <p>
-                Évidemment. Les syndicats sont là pour te représenter et te protéger dans ce qu’on appelle le dialogue
-                social, mais tu peux tout à fait être en grève sans avoir de lien avec un syndicat ou si il n’y a pas de
-                syndicat dans ton entreprise ou ton administration.
+                Évidemment. Les syndicats sont un outil à notre service, ils nous permettent de nous organiser 
+                collectivement pour défendre nos droits, mais il est tout à fait possible d'être en grève sans avoir 
```

Comment:
```suggestion
                collectivement pour défendre nos droits, mais il est tout à fait possible d’être en grève sans avoir 
```
…cette obsession des apostrophes courbes nous perdra.